data/Gaussian/basicGaussian16/water_hf_solvent_ipcm.com

```diff
@@ -0,0 +1,13 @@
+%Chk="water.chk"
+%Rwf="water.rwf"
+%NProcShared=2
+%Mem=4GB
+#p SP PBE1PBE/STO-3G SCF=(DIIS) Symmetry=(Tight) SCRF=(IPCM, Solvent=Toluene) EmpiricalDispersion=(GD3BJ) Integral=(Ultrafine) Population=(Regular) Density=(Current)
```

Comment:
I meant to make sure that all of the calculations run as Hartree-Fock and not DFT.

Also, in general, none of them need number of processors or memory specified...these calculations are tiny.

Reply:
Ah apologies re. the HF, I misunderstood.
Our calculation framework makes it difficult to not specify the number of CPUs and memory altogether, but I can set them to low numbers if that works (1 CPU/500MB). Likewise for %chk and %fchk, I can remove them from the input files after the calculation which should be fine (they'll still appear in the log file but hopefully that's not a problem?

Reply:
That's fine, none of it is really critical.  There were (still are?) some Gaussian inputs that do `geom=read` from the of course nonexistent checkpoint file and don't specify explicit coordinates, which is really annoying...

Reply:
Yep those are still there haha. I tried to re-run one to get the starting geometry out and was baffled for a good 5 mins why I couldn't get it to work